go-server-server/go/default.go

```diff
@@ -1229,7 +1230,29 @@ func ConfigVrouterVrfIdRoutesPatch(w http.ResponseWriter, r *http.Request) {
             failed = append(failed, r)
             continue
         }
-
+        kv, err := GetKVs(confdb.db_num, generateDBTableKey(confdb.separator, VNET_TB, vnet_id_str))
+        if err != nil {
+            WriteRequestError(w, http.StatusInternalServerError, "Internal service error", []string{}, "")
+            return
+        }
+        if adv_prefix, ok := kv["advertise_prefix"]; ok {
+            if adv_prefix == "true" {
+                prefix_len, _ := network.Mask.Size()
+                if isV4orV6(ip.String()) == 4 {
```

Comment:
Can we use this - `ip.To4()`

Reply:
The `isV4orV6` method is calling `ip.To4()`